alembic/env.py

```
@@ -12,12 +12,14 @@
 from alembic import context
 
 from app.core.config import get_settings
-from app.core.database import Base
+from app.models.base import Base
+from app.models import User
```

Ruff (0.11.9)

`app.models.User` imported but unused

Remove unused import: `app.models.User`

(F401)